source/index.ts

```diff
@@ -223,6 +223,25 @@ class Conf<T extends Record<string, any> = Record<string, unknown>> implements I
 		this.store = store;
 	}
 
+	/**
```

Comment:
The types here should limit it to only allow using `appendToArray` on a key defined to have an array value or unknown.

Reply:
I tried many, but can't figure it out. The best I could come up with was
```ts
appendToArray<Key extends keyof T, Index extends keyof any[]>(key: Key, value: T[Key][Index]): void;
```

Reply:
@sindresorhus I'm not able to make it, would like to know the solution please